src/types/state.ts

```diff
@@ -40,7 +40,9 @@ export interface DeviceStatus {
 		};
 	} & DeviceReportFields;
 	// TODO: Type the dependent entry correctly
-	dependent?: any;
+	dependent?: {
+		[key: string]: any;
+	};
```

Comment:
This is very slightly better and was necessary for `keyof DeviceStatus['dependent']` above to actually return a reasonable type